test/src/unit-cppapi-consolidation.cc

```diff
@@ -488,3 +489,52 @@ TEST_CASE(
   if (vfs.is_dir(array_name))
     vfs.remove_dir(array_name);
 }
+
+TEST_CASE(
+    "C++ API: Test consolidation that respects the current domain",
+    "[cppapi][consolidation][current-domain]") {
```

Comment:
Let's use `current_domain` as most of the other test cases related to current domain

Reply:
Forgot that :P